### ISSUE TITLE
Use semantics metadata instead of habot endpoint

### ISSUE THREAD

**Issue by Alpha200** [Alpha200/snips-openhab#14]

**Comment by nanosonde** [comment]
How do you plan to do this? 
Do you just want to directly use the content of the tags?

Example
```
  {
    "link": "http://192.168.192.2:8080/rest/items/GF_Office_Temperature",
    "state": "23.42",
    "stateDescription": {
      "pattern": "%.1f °C",
      "readOnly": false,
      "options": []
    },
    "editable": false,
    "type": "Number",
    "name": "GF_Office_Temperature",
    "label": "Temperatur",
    "category": "temperature",
    "tags": [
      "Measurement",
      "Temperature"
    ],
    "groupNames": [
      "GF_Office",
      "gTemperature"
    ]
  },
```

**Comment by Alpha200** [comment]
No, I discovered that you can get the relations of the Eclipse Smart Home Ontology via the items endpoint. Just add "synonyms,semantics" to the metadata parameter and you will get something like this:

```{
  "link": "http://syno:8080/rest/items/Temperature_Bedroom",
  "state": "UNDEF",
  "stateDescription": {
    "pattern": "%.1f °C",
    "readOnly": false,
    "options": []
  },
  "metadata": {
    "semantics": {
      "value": "Point_Measurement",
      "config": {
        "relatesTo": "Property_Temperature",
        "hasLocation": "schlafzimmer"
      }
    }
  },
  "editable": false,
  "type": "Number",
  "name": "Temperature_Bedroom",
  "label": "Temperatur",
  "category": "temperature",
  "tags": [
    "Measurement",
    "Temperature"
  ],
  "groupNames": [
    "schlafzimmer"
  ]
}
```

**Comment by nanosonde** [comment]
Thanks for the hint. Very interesting. Where did you find the documentation for this? I did not find anything in the REST API docs.

Do you plan to implement this in short-term?
UPDATE: Just found your work here: https://github.com/Alpha200/snips-openhab/commits/feature/semantics-api

**Comment by nanosonde** [comment]
Another question.

My item:
```
Number          GF_Office_Temperature          "Temperatur [%.1f °C]"<temperature>     (GF_Office, gTemperature) ["Measurement", "Temperature"] {channel="knx:device:bridge:sensor_ta63:GF_Office_Temperature"}
```
Just tries this request:
```
http://homeserver:8080/rest/items/GF_Office_Temperature?metadata=synonyms%2Csemantics
```
And got this response:
```
{
  "link": "http://homeserver:8080/rest/items/GF_Office_Temperature",
  "state": "23.1",
  "stateDescription": {
    "pattern": "%.1f °C",
    "readOnly": false,
    "options": []
  },
  "metadata": {
    "semantics": {
      "value": "Point_Measurement",
      "config": {
        "relatesTo": "Property_Temperature",
        "hasLocation": "GF_Office"
      }
    }
  },
  "editable": false,
  "type": "Number",
  "name": "GF_Office_Temperature",
  "label": "Temperatur",
  "category": "temperature",
  "tags": [
    "Measurement",
    "Temperature"
  ],
  "groupNames": [
    "GF_Office",
    "gTemperature"
  ]
}
```

The only German word is the German label "Temperatur". 
Did you find a way of getting the language specific translations fro OH for use with SNIPS entities injection?

**Comment by nanosonde** [comment]
Ok, I found the answer to my question here in your feature branch: https://github.com/Alpha200/snips-openhab/blob/9bd4882aa6411885b27ac292db1a789dc89f83f5/openhab.py#L73

**Comment by Alpha200** [comment]
Yes, I did not find a better solution. If you find a way to get it via the rest API let me know.

**Comment by Alpha200** [comment]
> Thanks for the hint. Very interesting. Where did you find the documentation for this? I did not find anything in the REST API docs.

I did not find any documentation to that. I've found a hint that you can get the metadata in this presentation: https://www.hft-stuttgart.de/Forschung/i_city/Handlungsfelder/Explorative-Projekte-i-city/Dokumente/praes9/de/ on slide 16.

**Comment by Alpha200** [comment]
Implemented in 620a2da
